addon-test-support/ilios-common/page-objects/course.js

```diff
@@ -32,7 +32,7 @@ export default create({
   header: {
     scope: '[data-test-course-header]',
     title: text('.editable'),
-    edit: clickable('.clickable'),
```

Comment:
Class shouldn't be used for test selectors anymore. This is a holdout from an old way. Please change to `[data-test-edit]`.

Reply:
Fixed.